ONSdigital/ras-collection-instrument
# Fix deprecated set output

# What and why?
This PR replaces set-output within the .github workflow file with environment variables to fix a deprecation with set-output. It also updates actions/auth to v2.

Pipfile has also been updated to resolve werkzeug vulnerability, flask 3 is included.

# How to test?
Check the github builds for any deprecation warnings and ensure everything functions as intended.

# Jira
https://jira.ons.gov.uk/secure/RapidBoard.jspa?rapidView=1493&projectKey=RAS&view=detail&selectedIssue=RAS-915
